core/src/main/scala/codesearch/core/search/Search.scala

```diff
@@ -49,12 +49,13 @@ trait Search {
             .toList
         } yield CSearchPage(results.sortBy(_.pack.name), entity.lists.size, ErrorResponse(""))
       }
-      case scala.util.Failure(exception) =>
-         {
-           val message = exception.getMessage
-           IO(CSearchPage(Seq.empty[Search.PackageResult], 0,
-             ErrorResponse(message.substring(0,1).toUpperCase + message.substring(1, message.size))))
-         }
+      case scala.util.Failure(exception) => {
+        val message = exception.getMessage
+        IO(
+          CSearchPage(Seq.empty[Search.PackageResult],
+                      0,
+                      ErrorResponse(message.substring(0, 1).toUpperCase + message.substring(1, message.size))))
```

Comment:
Use `.lenght` instead `.size` on strings.

Reply:
> Use .lenght instead .size on strings.

Does Scala have a linter that catches those things? (Also it's `.length`)

Reply:
@neongreen only IDE inspection support.
![image](https://user-images.githubusercontent.com/5605372/57177908-95d3cb80-6e82-11e9-999e-a94126919094.png)

Reply:
I meant a linter that we can run during CI.

I have just tried to run SonarQube on our code, but it didn't find the `message.size` thing :(